x-pack/plugins/security_solution/public/resolver/view/panels/node_events_of_type.test.tsx

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { createMemoryHistory, History as HistoryPackageHistoryInterface } from 'history';
+
+import { oneNodeWithPaginatedEvents } from '../../data_access_layer/mocks/one_node_with_paginated_related_events';
+import { Simulator } from '../../test_utilities/simulator';
+// Extend jest with a custom matcher
+import '../../test_utilities/extend_jest';
+import { urlSearch } from '../../test_utilities/url_search';
+
+// the resolver component instance ID, used by the react code to distinguish piece of global state from those used by other resolver instances
+const resolverComponentInstanceID = 'resolverComponentInstanceID';
+
+describe(`Resolver: when analyzing a tree with only the origin and paginated related events, and when the component instance ID is ${resolverComponentInstanceID}`, () => {
+  /**
+   * Get (or lazily create and get) the simulator.
+   */
+  let simulator: () => Simulator;
+  /** lazily populated by `simulator`. */
+  let simulatorInstance: Simulator | undefined;
+  let memoryHistory: HistoryPackageHistoryInterface<never>;
+
+  beforeEach(() => {
+    // create a mock data access layer
+    const { metadata: dataAccessLayerMetadata, dataAccessLayer } = oneNodeWithPaginatedEvents();
+
+    memoryHistory = createMemoryHistory();
+
+    // create a resolver simulator, using the data access layer and an arbitrary component instance ID
+    simulator = () => {
+      if (simulatorInstance) {
+        return simulatorInstance;
+      } else {
+        simulatorInstance = new Simulator({
+          databaseDocumentID: dataAccessLayerMetadata.databaseDocumentID,
+          dataAccessLayer,
+          resolverComponentInstanceID,
+          history: memoryHistory,
+          indices: [],
+        });
+        return simulatorInstance;
+      }
+    };
+  });
+
+  afterEach(() => {
+    simulatorInstance = undefined;
+  });
+
+  describe(`when the URL query string is`, () => {
+    beforeEach(() => {
+      memoryHistory.push({
+        search: urlSearch(resolverComponentInstanceID, {
+          panelParameters: { nodeID: 'origin', eventCategory: 'registry' },
+          panelView: 'nodeEventsInCategory',
+        }),
+      });
+    });
+    it('should show the load more data button', async () => {
+      await expect(
+        simulator().map(
+          () => simulator().testSubject('resolver:nodeEventsInCategory:loadMore').length
+        )
+      ).toYieldEqualTo(1);
+      await expect(
```

Comment:
Could you combine this `expect` with the one above it? That way the test will only pass if the 'load more' button is shown at the same time as 25 links.